app/src/androidTest/java/org/mozilla/fenix/ui/robots/BrowserRobot.kt

```diff
@@ -631,6 +631,24 @@ class BrowserRobot {
     }
 
     fun verifyPrefilledLoginCredentials(userName: String) {
+        var currentTries = 0
```

Comment:
Can we add a comment here along the lines of, sometimes the assertFails, so we retry the prefiling the login?

Reply:
Done !